pkg/components/controllers.go

```diff
@@ -175,7 +177,8 @@ func startIngressController(ctx context.Context, cfg *config.Config, kubeconfigP
 		return err
 	}
 
-	if err := assets.ApplyDeployments(ctx, apps, renderTemplate, renderParamsFromConfig(cfg, nil), kubeconfigPath); err != nil {
+	now := time.Now().Unix()
```

Comment:
Instead of repeating this pattern everywhere, maybe we want a factory for RenderParams to add the timestamp? If we already use renderParamsFromConfig everywhere we could just add it there.

If ApplyDeployments() added the annotation, we wouldn't even need to update the assets to include the template parameter.

Reply:
Hm, you are right, didnt realize that was a function we owned. Actually this is a pattern in library-go. Will do that and also rebase the PR to the latest changes in tests.